homeassistant/components/nibe_heatpump/climate.py

```diff
@@ -112,7 +112,12 @@ def _get(address: int) -> Coil:
 
         self._coil_current = _get(climate.current)
         self._coil_setpoint_heat = _get(climate.setpoint_heat)
-        self._coil_setpoint_cool = _get(climate.setpoint_cool)
+        try:
+            self._coil_setpoint_cool = _get(climate.setpoint_cool)
```

Comment:
Im thinking we should allow setpoint cool to be none. And use that to indicate support?

Reply:
OK sure, lets define it to be optional and check that instead of the boolean. 
Just tested and updated ;)